tests/installation/bootloader_uefi.pm

```diff
@@ -117,7 +119,7 @@ sub run {
         # random magic numbers
         send_key_until_needlematch('inst-onupgrade', 'down', 11, 3);
     }
-    else {
+    elsif (get_var('VERSION') !~ /agama/) {
```

Comment:
you might break the logic for another livecd, like tumbleweed, not sure why you need this change...

Reply:
Sometimes it can't match the boot-live-gnome in short time, https://openqa.opensuse.org/tests/3406032#step/bootloader_start/3  I add condition to avoid it to match boot-live-gnome.